README.md

```diff
@@ -128,19 +129,27 @@ Usage:
   rabtap -h|--help
   rabtap info [--api=APIURI] [--consumers] [--stats] [--filter=EXPR] [--omit-empty] 
               [--show-default] [--mode=MODE] [--format=FORMAT] [-knv]
+              [--cert-file=CERTFILE] [--key-file=KEYFILE] [--ca-file=CAFILE]
   rabtap tap EXCHANGES [--uri=URI] [--saveto=DIR] [--format=FORMAT] [-jknsv]
+                       [--cert-file=CERTFILE] [--key-file=KEYFILE] [--ca-file=CAFILE]
```

Comment:
we should name the options `--tls-key-file`, `--tls-cert-file` and `--tls-ca-file` to make it clear that these options are TLS related?